src/peoplemeasurement/serializers.py

```diff
@@ -41,13 +41,32 @@ class Meta:
         fields = '__all__'
 
 
-class AreaSerializer(serializers.ModelSerializer):
+class GeoSerializer(serializers.ModelSerializer):
+    sensor = serializers.SlugRelatedField(slug_field="objectnummer", queryset=Sensors.objects.all())
+
+    def get_validation_errors(self, errors=None) -> list[str]:
+        """
+        Returns full errors formatted as per requirements
+        """
+        default_errors = errors or self.errors  # default errors dict
```

Comment:
Deze commentaar zou je wat mij betreft weg kunnen laten, of anders formuleren als je denkt dat `default_errors` wat uitleg nodig heeft.